elasticsearch.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- hosts: "{{ hosts_prefix }}-tsuru-elasticsearch"
+  sudo: yes
+  sudo_user: root
+  tasks:
+    - name: Install required packages
+      apt:
+        pkg: "{{item}}"
+        state: installed
+        update_cache: "yes"
+      with_items:
+        - git
+        - python-dev
+        - openjdk-7-jre-headless
+        - python-pip
+    - name: Download ES
+      get_url:
+        url: https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.6.0.deb
+        dest: /tmp/elasticsearch-1.6.0.deb
+    - name: Install ES 1.6.0
+      apt:
+        deb: /tmp/elasticsearch-1.6.0.deb
+    - name: Start the service
+      service:
+        name: elasticsearch
+        state: started
+    - name: Git pull broker
+      git:
+        repo: https://github.com/RichardKnop/tsuru-elasticsearch-api.git
```

Comment:
We've had a conversation offline about this and think actually it would be better to build on the out of date [Elasticsearch API created by Globo](https://github.com/globocom/elastic-search-api) rather than use the one created for the spike. That one is perfectly fine for a spike, but if we are going to use it it would need a better README, clearer commit messages and ideally, tests, and building on the work that Globo have already done around this would put us in a better position.